commit fe36bf6f0dc1f0c1e927850dd78093cfe3b7aae2
Author: Albert Astals Cid <aacid@kde.org>
Date:   Sun Jul 28 00:07:33 2019 +0200

    Fix compile on CI
    
    src/workpackage/view.h:148:31: error: field m_scheduleActions has incomplete type QMap<QAction*, KPlato::Schedule*>

diff --git a/src/workpackage/view.h b/src/workpackage/view.h
index 6cae32b7..cd2fc4ea 100644
--- a/src/workpackage/view.h
+++ b/src/workpackage/view.h
@@ -24,6 +24,7 @@
 
 #include <KoView.h>
 
+#include <QMap>
 #include <QStackedWidget>
 
 class QPrinter;
